lib/views/desktop/receive/receive.dart

```diff
@@ -130,8 +130,8 @@ class ReceiveScreen extends StatelessWidget {
   Widget transferRejected() {
     return Consumer<ReceiveSharedState>(builder: (context, state, _) {
       return DTErrorUI(
-          errorTitle: 'The transfer was cancelled by the receiver.',
-          error: state.error != null ? state.error : '',
+          errorTitle: state.errorTitle,
+          error: state.error,
           errorMessage: state.errorMessage,
```

Comment:
Two questions:
1. do we need different widget for transferCancelled and transferRejected? Basically only messages are different
2. why transferCancelled has: errorMessage: state.errorMessage ?? UNKNOWN_ERROR and transferRejected: errorMessage: state.errorMessage only
3. the same error: state.error != null ? state.error : '', vs error: state.error,